src/main/java/org/terasology/fluid/system/FluidAuthoritySystem.java

```diff
@@ -52,6 +57,34 @@
     private FluidRegistry fluidRegistry;
     @In
     private InventoryManager inventoryManager;
+    
+    @In
+    private BlockManager blockManager;
+    private Block air;
+    
+    @In
+    private ExtraBlockDataManager extraDataManager;
+    private int flowIx;
+    @In
+    private ComponentSystemManager componentSystemManager;
+    private boolean flowingLiquidsEnabled;
+    
+    /**
+     * If one block is 1m across, the fluid units are litres. This works reasonably
+     * sensibly with the pre-existing container sizes in ManualLabor.
+     */
+    private static final float FLUID_PER_BLOCK = 1000;
```

Comment:
Since this is a constant should it be up by the top of the class instead?

Reply:
Checkstyle doesn't even complain about this. I've moved it anyway.

Reply:
Actually I think Checkstyle might just not be running for some reason. It's also not complaining about some other stuff that's definitely against the rules.

Reply:
I think I've gotten used to the IDE warnings mixing together Checkstyle and some IntelliJ recommendations, not even sure which one covers this. With the latest Jenkins rebuild we do get this nice check:

![image](https://user-images.githubusercontent.com/1063833/88690484-54e85800-d0c1-11ea-8771-3351240e4842.png)

Although it is still pretty mild and hidden, I think we can change the severity config, add gates, etc to make it look more alarming or block PRs. Another option is to have a bot make outright comments or something (that just might get spammy). Finally I *do* believe there's another level to the fancy Cloudbees Jenkins GitHub PR thing that can be enabled with a full GitHub app - that _may_ be outright line coverage richly outlined in the PR which would be amazing, but I'm not sure as I haven't had time yet to dig deeper.

Following the link that gets posted you get all the Checkstyle details, can even drill down to where this exact one is covered :-)

http://jenkins.terasology.io/teraorg/job/Terasology/job/Modules/job/F/job/Fluid/job/PR-19/3/checkstyle/packageName.-1493123546/fileName.1389089918/type.1614765908/

![image](https://user-images.githubusercontent.com/1063833/88690922-ce804600-d0c1-11ea-8d54-093268f05b2a.png)

So we might be in business, just needing to make it all more visible